moduleframework/tools/static_dockerfile_checks.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import print_function
+
+import os
+from avocado import Test
+from moduleframework import module_framework
+from moduleframework import dockerlinter
+
+class DockerInstructionsTests(module_framework.AvocadoTest):
+    """
+    :avocado: enable
+    :avocado: tags=sanity,rhel,fedora,docker,docker_instruction_test
+
+    """
+
+    dp = None
+
+    def setUp(self):
+        # it is not intended just for docker, but just docker packages are
+        # actually properly signed
+        self.dp = dockerlinter.DockerfileLinter()
+        if self.dp.dockerfile is None:
+            self.skip("Dockerfile was not found")
+
+    def tearDown(self, *args, **kwargs):
```

Comment:
when you change parent to ``Test`` you can remove this empty ``tearDown`` function